sdk/nodejs/resource.ts

```diff
@@ -1511,3 +1511,24 @@ export function pkgFromType(type: string): string | undefined {
     }
     return undefined;
 }
+
+
+/**
+ * The type assigned to the resource at construction.
```

Comment:
I think we could flesh this out a tiny bit:
```suggestion
 * The Pulumi type assigned to the resource at construction, of the form `package:module:name`.
```
or something like that? Just because there are so many "types" floating around.